spec/minimal_repo.rb

```diff
@@ -5,13 +5,7 @@ module MinimalRepo
   class << self
     DIRECTORIES = [
       '.git',
-      'config',
-      'hosts',
-      'genders',
-      'dhcp',
       'files',
```

Comment:
Actually I may be able to do this simply, let me just try it out and see if it looks nicer.